pkg/util/users/users.go

```diff
@@ -124,6 +153,22 @@ func (strategy DefaultUserSyncStrategy) alterPgUserSet(user spec.PgUser, db *sql
 	return nil
 }
 
+func (strategy DefaultUserSyncStrategy) alterPgUserRename(user spec.PgUser, db *sql.DB) error {
+	var query string
+
+	if user.Deleted {
```

Comment:
I find the naming here misleading:
if the user is deleted why do we actually trim the `RoleDeletionSuffix` from the `newName` ?
That reads non-obvious to me